keps/sig-node/127-user-namespaces/kep.yaml

```diff
@@ -0,0 +1,21 @@
+title: Support User Namespaces
+kep-number: 127
+authors:
+  - "@rata"
+  - "@giuseppe"
+  - "@mauriciovasquezbernal"
+owning-sig: sig-node
+participating-sigs: []
+status: provisional
+creation-date: 2021-11-02
+reviewers:
+  - "@mrunalp"
+  - "@SergeyKanzhelev"
+  - "@thockin"
+approvers:
+  - "@derekwaynecarr"
+feature-gates:
+  - name: UserNamespacesSupport
```

Comment:
Good point, switched to `UserNamespacesPhase1`

Reply:
This would add some complexity at the time when a phase2 FG lands and can be seen as a FG rename for the users who test the feature regarless of the phase.

Reply:
@neolit123 what do you suggest? To keep it as it was?

Reply:
yes, but it is just my opinion. ultimately the owning group (sig-node) decides.

Reply:
I suggest phase 1 will keep this name `UserNamespacesSupport`, phase 2 will introduce `UserNamespacesSupportWithVolumes` or something that can be enabled on top of `UserNamespacesSupport`

Reply:
That SGTM too, I've done the change locally.  I'll let @mrunalp chime-in and can push whatever we agree on, we can do it in the next PR too :)

Reply:
SGTM!